lib/Controller/RoomController.php

```diff
@@ -268,7 +269,9 @@ public function getSingleRoom(string $token): DataResponse {
 		try {
 			$isSIPBridgeRequest = $this->validateSIPBridgeRequest($token);
 		} catch (UnauthorizedException $e) {
-			return new DataResponse([], Http::STATUS_UNAUTHORIZED);
+			$response = new DataResponse([], Http::STATUS_UNAUTHORIZED);
+			$response->throttle();
```

Comment:
Does the delay in SIP bridge requests need to be reset as well? See comment about HPB.

Reply:
As per above I would like to not do this.